usaspending_api/references/models.py

```diff
@@ -144,13 +146,15 @@ class SubtierAgency(models.Model):
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
     subtier_code = models.CharField(max_length=4, blank=True, null=True, verbose_name="Sub-Tier Agency Code")
+    abbreviation = models.CharField(max_length=150, blank=True, null=True, verbose_name="Agency Abbreviation")
```

Comment:
No, there's no need to title it like that, because to access it you still need `awarding_agency__subtier_agency__abbreviation` or `toptier_agency__abbreviation`, just like `name` so there is no confusion there. Generally, there's no need to have `MODELNAME_FIELDNAME`, it's just on some fields due to legacy code